TimezoneSelect/timezones.json

```diff
@@ -0,0 +1,1450 @@
+[
```

Comment:
Same as for `country-options.tsx`. I know it will take some time to format the file :) What's helping me in such situations is multiple cursor feature of VSC.

Reply:
If we keep the consistent format across all select boxes then we can also get rid of extra type — `Timezone`.